src/main/features/linux/mprisService.js

```diff
@@ -0,0 +1,60 @@
+const mpris = require('mpris-service');
+
+class MprisService {
+
+    constructor() {
+        this.player = mpris({
+            name: 'gpmdp',
+            identity: 'Google Play Music Desktop Player',
+            supportedInterfaces: ['player']
+        });
+
+        this.listeners();
+    }
+
+    listeners() {
```

Comment:
More of a style than anything else but "private" methods I.e. methods that are only called internally.  Can they start with an `_` :+1: 

So this would be `_listeners()`